### ISSUE TITLE
Orden de resultado de búsqueda

### ISSUE THREAD

**Issue by SebaHagedorn** [DiegoEmilio01/IIC3413#20]
Hola! solo para confirmar que estoy teniendo buenos resultados, cuando se imprime el resultado de la búsqueda, los records pueden estar desordenados o no? Gracias de antemano.

**Comment by cirojas** [comment]
sí, pueden estar desordenados, ya que dentro de una hoja y sus overflow pages no se mantiene un orden.

**Comment by SebaHagedorn** [comment]
Gracias
